tests/conftest.py

```diff
@@ -199,6 +200,7 @@ def db_seed():
         long_description="Pharmacists are healthcare practitioners who check and distribute drugs to doctors for medication that had been prescribed to patients. They advise patients and health care providers on the selection, dosages, interactions, and side effects of medications. They might work in a  hospital or retail store."
     )
     jobs = [job1, job2, job3, job4, job5, job6, job7, job8, job9, job10, job11, job12, job13, job14, job15]
+    db.session.execute("ALTER SEQUENCE jobs_id_seq RESTART WITH 1")
```

Comment:
Reset primary key of id every time the DB is populated. Important for testing /api/v1/jobs/<id> endpoint.